### PR TITLE
refactor: 🔨  refactor code and 💄  modify styles in personal home page

### PR DESCRIPTION
**Pull request by sherotree**
## What type of PR is this?

- [ ] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [x] Refactor
- [x] Style
- [ ] Chore

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [x] Yes(screenshot is required)
- [ ] No

## Special notes for your reviewer:
1. modify personal home page when route change orgName
2. extends some config-page components like Text  Title
3. update TableGroup for title(use Text take place of Title) to jumpOut
4. update some styles

### CODE REVIEW

File: shell/app/config-page/components/dropdown-select/dropdown-select.tsx [hunk 1]
```diff
@@ -70,15 +70,15 @@ const DropdownSelect = (props: CP_DROPDOWN_SELECT.Props) => {
...
+          map(options, (item: CP_DROPDOWN_SELECT.IOptionItem, index: number) => {
             // 前端搜索
             if (!item.label.toLowerCase().includes(filterValue)) {
               return null
             }
 
             return (
               <Menu.Item
-                key={item.label}
+                key={`${String(index)}`}
```
Comment (daskyrk) [inline shell/app/config-page/components/dropdown-select/dropdown-select.tsx:81]: just `String(index)` or "`${index}`"

File: shell/app/config-page/components/dropdown-select/dropdown-select.tsx [hunk 2]
```diff
@@ -105,8 +105,9 @@ const DropdownSelect = (props: CP_DROPDOWN_SELECT.Props) => {
         <Menu.Divider key='divider2' />
         {
           quickSelect.length > 0 ?
-            map(quickSelect, (item) => (
+            map(quickSelect, (item, index) => (
               <Menu.Item
+                key={`${String(index)}`}
```
Comment (daskyrk) [inline shell/app/config-page/components/dropdown-select/dropdown-select.tsx:110]: same as above

File: shell/app/config-page/components/empty-holder/empty-holder.tsx [hunk 3]
```diff
@@ -20,20 +20,29 @@ import './empty-holder.scss';
 export const EmptyHolder = (props: CP_EMPTY_HOLDER.Props) => {
   const { props: configProps } = props;
   const {
+    visible = false,
```
Comment (laojun) [inline shell/app/config-page/components/empty-holder/empty-holder.tsx:23]: default visible = true

File: shell/app/config-page/components/title/title.scss [hunk 4]
```diff
@@ -40,6 +40,14 @@
     .circle {
       border-radius: 50%;
     }
+
+    .erda-logo {
```
Comment (laojun) [inline shell/app/config-page/components/title/title.scss:44]: do not use specifically name in config page

File: shell/app/config-page/components/container/container.scss [hunk 5]
```diff
@@ -4,6 +4,12 @@
       display: flex;
       flex-direction: column;
 
+      &.space-none {
+        > * {
+          margin-left: 0;
```
Comment (laojun) [inline shell/app/config-page/components/container/container.scss:9]: here is 'margin-top'

File: shell/app/config-page/components/image/image.scss [hunk 6]
```diff
@@ -34,4 +34,10 @@
     border-radius: 50%;
   }
 
+  &.erda-logo {
+    width: 32px;
+    height: 32px;
+    padding: 4px;
+    background-color: $color-primary;
+  }
```
Comment (laojun) [inline shell/app/config-page/components/image/image.scss:42]: useless

**Review by laojun — APPROVED** [review]

**Pull request merged**